deel-3-webpagina/styles/styles.css

```diff
@@ -1 +1,86 @@
-/*Schrijf hier jouw css*/
\ No newline at end of file
+/*Schrijf hier jouw css*/
+
+/*Schrijf hier jouw css*/
+
+body {
+    background-color: #F2F3D9;
+    margin: 0;
+}
+
+.navbar {
+    background-color: #030027;
+    color: #fff;
+    padding: 40px;
+}
+
+.navbar .container {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+
+.navbar a {
```

Comment:
deze is overbodig als je de .navbar een kleur meegeeft en de body een font familiy